tracing-subscriber/src/registry/fmt_layer.rs

```diff
@@ -0,0 +1,197 @@
+use crate::{
+    fmt::{format, FormatEvent, FormatFields, MakeWriter},
+    layer::{Context, Layer},
+    registry::{Extensions, LookupMetadata, LookupSpan, Registry, SpanData, SpanRef},
+};
+use ansi_term::{Color, Style};
+use humantime;
+use std::{
+    any::type_name,
```

Comment:
this breaks the build on Rust 1.34.0

Reply:
yep, removed. it's useful for debugging _what_ concrete type was instantiated.